### PR TITLE
feat:added guide for multimeter instrument

### LINKED ISSUE #971: No documentation for multimeter activity
**Actual Behaviour**

There is no documentation showing how to use the multimeter in PSLab.

**Expected Behaviour**

There should be a basic documentation stating how to use the multimeter

**LogCat for the issue**

Not applicable

**Screenshots of the issue**

Not applicable

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by abhinavraj23**
Fixes #971 

Changes:Added a guide for multimeter instrument

Screenshots for the change: 
multimeter_guide

APK for testing: [Compress the *.apk file into a rar or zip file and upload it here]
[app-debug.apk.zip](https://github.com/fossasia/pslab-android/files/2077621/app-debug.apk.zip)

### CODE REVIEW

**Comment by harsh-2711** [comment]
@abhinavraj23 use colors that are used in previous merged PRs and refrain from making dots at the edges. Also draw the wires from the center and not from the end.

**Comment by abhinavraj23** [comment]
@Vikum94 @harsh-2711 All changes done :+1:

**Comment by harsh-2711** [comment]
@abhinavraj23  The second bullet in the image isn't proper. I mean what should be connected to pin ID1,ID2... Just specify power source there also. Also the section above image is for description and so don't add steps there. Please specify them below image.

**Comment by abhinavraj23** [comment]
@harsh-2711 The SQ channels should be connected to it, I will update it :+1:

**Comment by abhinavraj23** [comment]
Changes done !

**Comment by abhinavraj23** [comment]
@harsh-2711 @CloudyPadmal @Avjeet please review

File: app/src/main/java/org/fossasia/pslab/activity/MultimeterActivity.java [hunk 1]
```diff
@@ -134,4 +145,42 @@ public void onClick(View v) {
...
     }
+
+    @SuppressLint("ResourceType")
+    public void howToConnectDialog(String title, String intro, int iconID, String desc) {
+        try {
+            final AlertDialog.Builder builder = new AlertDialog.Builder(this);
+            LayoutInflater inflater = getLayoutInflater();
+            View dialogView = inflater.inflate(R.layout.custom_dialog_box, null);
+            final SharedPreferences settings = this.getSharedPreferences(PREFS_NAME, 0);
+            Boolean skipMessage = settings.getBoolean("skipMessage", false);
```
Comment (CloudyPadmal) [inline app/src/main/java/org/fossasia/pslab/activity/MultimeterActivity.java]: I was going through all PRs related to dialog boxes and it seems like everyone's using too abstract variable name for shared preference boolean. Make this instrument specific. `settings.getBoolean("<skipMultimeterMessage>", false);`
I'll request this change in other PRs as well.

File: app/src/main/java/org/fossasia/pslab/activity/MultimeterActivity.java [hunk 2]
```diff
@@ -134,4 +145,42 @@ public void onClick(View v) {
...
 
     }
+
+    @SuppressLint("ResourceType")
+    public void howToConnectDialog(String title, String intro, int iconID, String desc) {
+        try {
+            final AlertDialog.Builder builder = new AlertDialog.Builder(this);
+            LayoutInflater inflater = getLayoutInflater();
+            View dialogView = inflater.inflate(R.layout.custom_dialog_box, null);
+            final SharedPreferences settings = this.getSharedPreferences(PREFS_NAME, 0);
```
Comment (CloudyPadmal) [inline app/src/main/java/org/fossasia/pslab/activity/MultimeterActivity.java]: Use `Private` mode in sharedpreference and try to use a common `sharedPreferences` as this app is not too big to have multiple sharedpreferences.

**Comment by abhinavraj23** [comment]
@CloudyPadmal All changes done!

**Pull request merged**